pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -487,6 +513,16 @@ func estimateGracefulTerminationForPods(kubeClient clientset.Interface, ns strin
 	return estimate, nil
 }
 
+func unstructuredToPod(obj *runtime.Unstructured) (*api.Pod, error) {
```

Comment:
Probably I missed something here. I thought the fed controller also has an instance of kube clientset, can you pass the CoreInterface of that one?